src/common/components/footnote/footnote.less

```diff
@@ -1,5 +1,10 @@
 .jdn__footnote {
-    font-size: 12px;
+    font-size: 14px;

```

Comment:
this change will affect markup in many components. footnote is a common class, made special for 12px fonts. If you need another font styles, you can replace `<Footnote />` with `<<Typography.Text />` imported from antd. all typography components can be found here: https://4x.ant.design/components/typography/#components-typography-demo-text